### PR TITLE
Style: Fix- Improve links to JBX projects

### PR DESCRIPTION
**Pull request by sticklo**
This PR removes the tooltip on project handles  and links to projects straight from the handle. 
There's a blue highlight and underline on hover of project handle and the links opens up in the same tab.

##Before-PR
160887662-21da825f-15da-44aa-910a-99ac3f0925ad

##After-PR
fix project link jbx

### CODE REVIEW

File: src/components/v1/shared/V1ProjectHandle.tsx [hunk 1]
```diff
@@ -14,24 +13,21 @@ export default function V1ProjectHandle({
   style?: CSSProperties
   link?: boolean
 }) {
+  const [hover, setHover] = useState(false)
```
Comment (johnnyd-eth) [inline src/components/v1/shared/V1ProjectHandle.tsx:16]: **Suggestion:** think we can avoid user hover states by using the class `hover-action`

**Review by johnnyd-eth — CHANGES_REQUESTED** [review]
Hey @sticklo, thanks for taking this on, great job! Left one comment above on the hover states.

Few more points:
- Let's make the non-hover state black just as they are now
- Let's remove the link icon altogether

File: src/components/v1/shared/V1ProjectHandle.tsx (hunk 1 above)
Comment (sticklo) [reply src/components/v1/shared/V1ProjectHandle.tsx:16]: alright I'm on it

**Comment by johnnyd-eth** [comment]
Nice @sticklo just one more thing: let's make the projects and normal handles have the same font-weights. Then we 🚀

Screen Shot 2022-04-07 at 4 15 33 pm

**Comment by sticklo** [comment]
from the image above,  what's the project?  and what is the normal handle?  + which of the elements are we applying the same font weight to?

**Comment by johnnyd-eth** [comment]
> from the image above, what's the project? and what is the normal handle? + which of the elements are we applying the same font weight to?

Ones starting with @ like '@peel' are projects. We want to keep the font weights as they are currently, which are the less thick ones

**Comment by sticklo** [comment]
okay so django.eth, drgorrila.eth are the normal handles we want to apply the same font weight as the projects for ?

**Comment by johnnyd-eth** [comment]
> okay so django.eth, drgorrila.eth are the normal handles we want to apply the same font weight as the projects for ?

Other way round. Keep the font weights as they currently are on the live site which is the less weighted (same as jango.eth, etc).

**Comment by sticklo** [comment]
I apologize for the back and forth but I still do not get it. Perhaps you could show me what you mean better over a screen share later in the day ?

**Comment by aeolianeth** [comment]
Hey @sticklo @johnnyd-eth sorry to barge in here, but I'd like to propose an alternative solution. I've prepared a patch to illustrate my idea. The patch does the following:

- creates CSS utility classes for the text styles.
- ensures the font weights for Juicebox project handles (for example `@peel`, `@juicebox`) have the same font weight as ENS names and addresses (for example `aeolian.eth`).
- uses the `style` prop of V1ProjectHandle to give devs the ability to override the link styles if necessary.
- removes unused `link` prop from V1ProjectHandle

Here is a demo of the changes:

https://user-images.githubusercontent.com/94939382/162349695-300da47a-69ad-4848-8c7d-3fb5a3234194.mp4

How to apply the patch:

1. Copy this diff to your clipboard.
   Hover over the diff to reveal a "Copy" button in the top-right corner of the diff.
3. In your terminal, apply the diff with:

Please review and apply the patch when you get a chance!

If you have any trouble with this, let me know and I'll commit the changes for you if you agree with them.

   ```
   pbpaste | git apply
   ```

```diff
diff --git a/src/components/Landing/Payments.tsx b/src/components/Landing/Payments.tsx
index 6571ae43..1daa7223 100644
--- a/src/components/Landing/Payments.tsx
+++ b/src/components/Landing/Payments.tsx
@@ -55,7 +55,7 @@ export default function Payments() {
                   style={{ color: colors.text.action.primary, fontWeight: 500 }}
                 >
                   {e.project?.id && (
-                    <V1ProjectHandle projectId={e.project.id} link />
+                    <V1ProjectHandle projectId={e.project.id} />
                   )}
                 </div>
                 <div
diff --git a/src/components/shared/Mod.tsx b/src/components/shared/Mod.tsx
index 344e3e2f..e962a4de 100644
--- a/src/components/shared/Mod.tsx
+++ b/src/components/shared/Mod.tsx
@@ -43,7 +43,6 @@ export default function Mod({
               <div style={{ fontWeight: 500 }}>
                 {(mod as PayoutMod).projectId ? (
                   <V1ProjectHandle
-                    link
                     projectId={(mod as PayoutMod).projectId as BigNumber}
                   />
                 ) : (
diff --git a/src/components/shared/ProjectTokenBalance.tsx b/src/components/shared/ProjectTokenBalance.tsx
index c1377fbc..c8023054 100644
--- a/src/components/shared/ProjectTokenBalance.tsx
+++ b/src/components/shared/ProjectTokenBalance.tsx
@@ -50,7 +50,6 @@ export default function ProjectTokenBalance({
       {!hideHandle && (
         <V1ProjectHandle
           style={{ color: colors.text.tertiary }}
-          link={true}
           projectId={projectId}
         />
       )}
diff --git a/src/components/shared/formItems/ProjectPayoutMods.tsx b/src/components/shared/formItems/ProjectPayoutMods.tsx
index 2dec3e35..7e7cf1c9 100644
--- a/src/components/shared/formItems/ProjectPayoutMods.tsx
+++ b/src/components/shared/formItems/ProjectPayoutMods.tsx
@@ -178,7 +178,7 @@ export default function ProjectPayoutMods({
                     }}
                   >
                     <span style={{ cursor: 'pointer' }}>
-                      <V1ProjectHandle link projectId={mod.projectId} />
+                      <V1ProjectHandle projectId={mod.projectId} />
                     </span>
                   </div>
                 </Col>
diff --git a/src/components/v1/V1Project/ProjectActivity/TapEventElem.tsx b/src/components/v1/V1Project/ProjectActivity/TapEventElem.tsx
index 9d609435..c2fe245a 100644
--- a/src/components/v1/V1Project/ProjectActivity/TapEventElem.tsx
+++ b/src/components/v1/V1Project/ProjectActivity/TapEventElem.tsx
@@ -101,7 +101,7 @@ export default function TapEventElem({
             <div style={{ fontWeight: 500 }}>
               {e.modProjectId?.gt(0) ? (
                 <span>
-                  <V1ProjectHandle link projectId={e.modProjectId} />
+                  <V1ProjectHandle projectId={e.modProjectId} />
                 </span>
               ) : (
                 <FormattedAddress address={e.modBeneficiary} />
diff --git a/src/components/v1/shared/V1ProjectHandle.tsx b/src/components/v1/shared/V1ProjectHandle.tsx
index 35157eb5..29df1c5d 100644
--- a/src/components/v1/shared/V1ProjectHandle.tsx
+++ b/src/components/v1/shared/V1ProjectHandle.tsx
@@ -6,16 +6,18 @@ import { Link } from 'react-router-dom'
 export default function V1ProjectHandle({
   projectId,
   style,
-  link,
 }: {
   projectId: BigNumberish
   style?: CSSProperties
-  link?: boolean
 }) {
   const handle = useHandleForProjectId(projectId)
 
   return (
-    <Link to={`/p/${handle}`} className="hover-action">
+    <Link
+      to={`/p/${handle}`}
+      style={{ fontWeight: 500, ...style }}
+      className="text-primary hover-text-action-primary hover-text-decoration-underline"
+    >
       @{handle}
     </Link>
   )
diff --git a/src/styles/utilities.scss b/src/styles/utilities.scss
index 8a9a07b5..3a5db796 100644
--- a/src/styles/utilities.scss
+++ b/src/styles/utilities.scss
@@ -1,6 +1,26 @@
+.text-primary {
+  color: var(--text-primary);
+}
 .hover-text-secondary {
   &:hover {
     color: var(--text-secondary) !important;
     transition: color 250ms;
   }
 }
+
+.hover-text-action-primary {
+  transition: unset;
+
+  &:hover {
+    color: var(--text-action-primary);
+  }
+}
+
+.hover-text-decoration-underline {
+  transition: unset;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+

```

**Comment by sticklo** [comment]
wow thank you so much @aeolianeth. I think this is fine. I had some errors trying to apply the diff you created for me, however I have manually applied all of your changes. Thank you for looking out.

**Comment by johnnyd-eth** [comment]
Nice @sticklo and thanks for jumping in @aeolianeth . Will merge when Fleek completes

**Comment by sticklo** [comment]
thank you @johnnyd-eth, @aeolianeth

**Review by johnnyd-eth — APPROVED** [review]

**Pull request merged**